lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer_test.go

```diff
@@ -38,12 +38,50 @@ import (
 	"magma/orc8r/cloud/go/services/configurator"
 	configurator_storage "magma/orc8r/cloud/go/services/configurator/storage"
 	configurator_test_init "magma/orc8r/cloud/go/services/configurator/test_init"
+	orc8rModels "magma/orc8r/cloud/go/services/orchestrator/obsidian/models"
 	"magma/orc8r/cloud/go/sqorc"
 	"magma/orc8r/cloud/go/storage"
 	"magma/orc8r/cloud/go/syncstore"
 	"magma/orc8r/lib/go/protos"
 )
 
+func TestListSuciProfiles(t *testing.T) {
+	lte_test_init.StartTestService(t)
+	configurator_test_init.StartTestService(t)
+	storeReader, _ := initializeStore(t)
+
+	servicer := servicers.NewSubscriberdbServicer(subscriberdb.Config{DigestsEnabled: false}, storeReader)
+
+	err := configurator.CreateNetwork(context.Background(), configurator.Network{
+		ID:          "nt1",
+		Type:        lte.NetworkType,
+		Name:        "foobar",
+		Description: "Foo Bar",
+		Configs: map[string]interface{}{
+			lte.CellularNetworkConfigType: lte_models.NewDefaultTDDNetworkConfig(),
+			orc8r.NetworkFeaturesConfig:   orc8rModels.NewDefaultFeaturesConfig(),
+			orc8r.DnsdNetworkType:         orc8rModels.NewDefaultDNSConfig(),
+		},
+	}, serdes.Network)
+	assert.NoError(t, err)
+
+	id := protos.NewGatewayIdentity("hw1", "nt1", "g1")
+	ctx := id.NewContextWithIdentity(context.Background())
+
+	expectedProtos := []*lte_protos.SuciProfile{
+		{
+			HomeNetworkPublicKeyIdentifier: 255,
+			HomeNetworkPublicKey:           []byte("\x12\x12\x12\x12"),
+			HomeNetworkPrivateKey:          []byte("\x12\x12\x12\x12"),
+			ProtectionScheme:               lte_protos.SuciProfile_ProfileA,
+		}}
```

Comment:
Nit: move the final curly brace to its own line

Reply:
done